Source/WebKit/UIProcess/ios/forms/WKDatePickerPopoverController.mm

```diff
@@ -38,38 +38,35 @@
 @interface WKDatePickerPopoverView : UIView
 
 @property (readonly, nonatomic) UIDatePicker *datePicker;
+@property (readonly, nonatomic) UIDatePicker *calendarView;
```

Comment:
I don't think this declaration is necessary? From a quick scan, I didn't see any code outside this class which accesses `calendarView`.

(also, the type is wrong)